pkgs/build-support/cc-wrapper/cc-wrapper.sh

```diff
@@ -79,6 +79,18 @@ if [ "$NIX_ENFORCE_PURITY" = 1 -a -n "$NIX_STORE" ]; then
     params=("${rest[@]}")
 fi
 
+if test -n "@libcxx@"; then
+    NIX_CFLAGS_COMPILE="$NIX_CFLAGS_COMPILE -isystem@libcxx@/include/c++/v1"
```

Comment:
Perhaps @copumpkin or @joelteon can comment? I didn't add this, but I know there was some trickiness with `libc++` that required the extra `NIX_SKIP_CXX[ABI]` variables. Though I suppose they could be merged into the `libc++` derivation.